extension/build.gradle

```diff
@@ -200,6 +200,7 @@ task prepareForBetaRelease() {
 
 task buildJdtlsPlugin(type: CrossPlatformExec) {
   workingDir file('./jdtls.ext')
+  environment 'MAVEN_OPTS', '-Djdk.xml.totalEntitySizeLimit=1000000 -Djdk.xml.maxGeneralEntitySizeLimit=1000000'
```

Comment:
https://github.com/microsoft/vscode-java-dependency/pull/886/files
consider setting the limit to 0